franklin/builder/helpers.py

```diff
@@ -33,4 +40,30 @@ def make_rest_post_call(url, headers, body):
     if response is not None:
         if not status.is_success(response.status_code):
             logger.warn('Bad POST response code of %s', response.status_code)
+            print('message', response.text)
+    else:
+        logger.error('POST response was None. This shouldn\'t happen')
+        response = requests.Response()
+        response.status_code=500
     return response
+
+def generate_ssh_keys():
+    # TODO - use https://pypi.python.org/pypi/pycrypto here after 2.7 comes out
+    return (None, None)
+
+
+class GithubOnly(permissions.BasePermission):
+    """
+    Security Check for certain API endpoints that are only called by Github.
```

Comment:
[This should be on the same line as the beginning](https://www.python.org/dev/peps/pep-0008/#documentation-strings) `"""`